projects/RPi/devices/RPi4/options

```diff
@@ -17,3 +17,5 @@
   # Kernel to use. values can be:
   # default:  default mainline kernel
     LINUX="rpi4-5.4"
+
+LIBRETRO_CORES="${LIBRETRO_CORES// melonds /}"
```

Comment:
This shouldn't be needed. melonds only is compiled for x86_64. RPi4 compiles for ARM atm so is more or less a no-op.

Reply:
I see now this should be defined in distribution/lakka/option, the one in master branch actually ignores RPi4 trying to compile almost everything, I was forced to remove melonds as compile process fails due to inconsistent repo for melonds.